api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -59,6 +78,20 @@ public BillingProjectBufferService(
     this.clock = clock;
     this.fireCloudService = fireCloudService;
     this.workbenchConfigProvider = workbenchConfigProvider;
+    View view = View.create(
```

Comment:
Can we do all this in one place, so we're not cluttering up individual classes being instrumented? If not, can I bet we can still use adapter services.

Reply:
I think we would have a `registerView` method or something? (`registerSignal` maybe? To be system agnostic?)